### PR TITLE
New global flags: avif, jpegxl, brotli, lzip + cleanup

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/qa

### CODE REVIEW

File: profiles/use.desc [hunk 1]
```diff
@@ -172,8 +175,9 @@ livecd - !!internal use only!! DO NOT SET THIS FLAG YOURSELF!, used during livec
 llvm-libunwind - Use sys-libs/llvm-libunwind instead of sys-libs/libunwind
 lm-sensors - Add linux lm-sensors (hardware sensors) support
 lua - Enable Lua scripting support
-lzma - Support for LZMA (de)compression algorithm
 lz4 - Enable support for lz4 compression (as implemented in app-arch/lz4)
+lzip - Enable support for lzip compression
```
Comment (thesamesam) [inline profiles/use.desc:179]: Why not (de) here too?

File: profiles/use.desc [hunk 2]
```diff
@@ -30,6 +31,7 @@ bindist - Flag to enable or disable options for prebuilt (GRP) packages (eg. due
 blas - Add support for the virtual/blas numerical library
 bluetooth - Enable Bluetooth Support
 branding - Enable Gentoo specific branding
+brotli - Enable Brotli compression support
```
Comment (thesamesam) [inline profiles/use.desc:34]: (De)?

**Review by thesamesam — APPROVED** [review]
Thank you!

File: profiles/use.desc (hunk 1 above)
Comment (mgorny) [reply profiles/use.desc:179]: Sounds like an overkill to me.

Comment (thesamesam) [reply profiles/use.desc:179]: Then why are we doing it at all for other formats? Clean it up in a new commit?

File: profiles/use.desc [hunk 3]
```diff
@@ -172,8 +175,9 @@ livecd - !!internal use only!! DO NOT SET THIS FLAG YOURSELF!, used during livec
 llvm-libunwind - Use sys-libs/llvm-libunwind instead of sys-libs/libunwind
 lm-sensors - Add linux lm-sensors (hardware sensors) support
 lua - Enable Lua scripting support
-lzma - Support for LZMA (de)compression algorithm
 lz4 - Enable support for lz4 compression (as implemented in app-arch/lz4)
+lzip - Enable support for lzip compression
+lzma - Support for LZMA (de)compression algorithm
```
Comment (thesamesam) [inline profiles/use.desc:180]: If it's overkill, you should remove it here too.

File: profiles/use.desc (hunk 2 above)
Comment (mgorny) [reply profiles/use.desc]: Have it your way.

Comment (thesamesam) [reply profiles/use.desc]: I don't care about having it, the issue is you're being inconsistent, just strip it entirely or add the prefix everywhere.

**Comment by ulm** [comment]
I've no strong opinion about this, but maybe the descriptions should be consistent? With `(de)` is slightly more accurate, without is slightly shorter.

**Comment by thesamesam** [comment]
Yes, I genuinely don't care as to if we have it or not, I only care for the consistency. Toss a coin :)

**Comment by mgorny** [comment]
Sigh, done.

**Pull request closed without merging**